cmake/MujocoDependencies.cmake

```diff
@@ -149,24 +149,30 @@ target_include_directories(
 target_compile_options(qhullstatic_r PRIVATE ${MUJOCO_MACOS_COMPILE_OPTIONS})
 target_link_options(qhullstatic_r PRIVATE ${MUJOCO_MACOS_LINK_OPTIONS})
 
-set(tinyxml2_BUILD_TESTING OFF)
-findorfetch(
-  USE_SYSTEM_PACKAGE
-  OFF
-  PACKAGE_NAME
-  tinyxml2
-  LIBRARY_NAME
-  tinyxml2
-  GIT_REPO
-  https://github.com/leethomason/tinyxml2.git
-  GIT_TAG
-  ${MUJOCO_DEP_VERSION_tinyxml2}
-  TARGETS
-  tinyxml2
-  EXCLUDE_FROM_ALL
-)
-target_compile_options(tinyxml2 PRIVATE ${MUJOCO_MACOS_COMPILE_OPTIONS})
-target_link_options(tinyxml2 PRIVATE ${MUJOCO_MACOS_LINK_OPTIONS})
+option(TINYXML2_USE_SYSTEM_PACKAGE "Use the system's tinyxml2" OFF)
+if(TINYXML2_USE_SYSTEM_PACKAGE)
+  find_package(tinyxml2 REQUIRED)
+  add_library(tinyxml2 ALIAS tinyxml2::tinyxml2)
+else()
```

Comment:
Instead of having an if, could it make sense to just pass `${TINYXML2_USE_SYSTEM_PACKAGE}` as `USE_SYSTEM_PACKAGE` argument, call `target_compile_options` and `target_link_options` only if `NOT TINYXML2_USE_SYSTEM_PACKAGE`, and in the rest of the CMake code (so in https://github.com/google-deepmind/mujoco/blob/50de1b3c2c62a42b394167b853cba9813f6b99c7/CMakeLists.txt#L143, https://github.com/google-deepmind/mujoco/blob/50de1b3c2c62a42b394167b853cba9813f6b99c7/src/experimental/usd/CMakeLists.txt#L101) refer to `tinyxml2::tinyxml2` ?